ImageD11/sinograms/tensor_map.py

```diff
@@ -1,6 +1,10 @@
 import os
 
 import numba
+# ignore Numba dot product performance warnings?
+import warnings
+warnings.simplefilter('ignore', category=numba.core.errors.NumbaPerformanceWarning)
```

Comment:
I would like to understand why it gives the warning

Reply:
If we suppress the warning we should create an issue to come back to it sometime in the future. Like this I am afraid we lose all of the feedback on numba compilation for anything importing our code...